### ISSUE TITLE
Windows update analyzer

### ISSUE THREAD

**Issue by tomchop** [google/timesketch#950]

**Comment by jaegeral** [comment]
What exactly would such an analyzer do? See if a system as updated?

**Comment by tomchop** [comment]
Yes, I think this was the idea when I wrote this. I'll leave a post-it to past Tomchop to be more explicit next time.

**Comment by berggren** [comment]
We have suggested queries for this now that will cover a lot of these. The session analyzers are not maintained so closing this issue.
